### ISSUE TITLE
Ignore some types of files like .DS_Store

### ISSUE THREAD

**Issue by Ilyes512** [Ilyes512/boilr#16]

**Comment by philoserf** [comment]
I believe it is the responsibility of the contributor to maintain a global gitignore (often `~/.gitignore_global`) that covers the OS and Editor they are using.

Then a project can maintain a project specific `.gitignore`.

**Comment by Ilyes512** [comment]
The problem is not with git. What happend was that I opened/viewed the cached template in finder. The location that boilr copies your added templates to. When you finally try to used it. It sees the dot file and probably it contained {{ and }}.

I shouldn’t have opened the location in finder anyways.

**Comment by philoserf** [comment]
The answer still could sit in your ~/.gitignore rather than in every project.

**Comment by Ilyes512** [comment]
Like I said...it's got nothing todo with git. I think the confusion comes from using the word "ignore".

When a user add's a template to boilr it copies the template to `~/.config/boilr/templates/yourtemplate`. So what I did was open that dir with `Finder` (are you familiar with or a user of MacOS?). Finder created a `.DS_Store` in that directory. Once I tried to use the template it tried to parse every file in that folder including the `.DS_Store`. It returned me an error instead, because the `.DS_Store` contained `{{` and `}}`.

So it's not about not adding committing the file to a repo. In fact the templates in `~/.config/boilr/template` don't contain a `.git` folder (while the original template might have).
